modules/ROOT/pages/_partials/_page-index.adoc

```diff
@@ -261,6 +256,126 @@ endif::xref--pfx-sgw[]
+:bootstrap-schema--xref--api-admin-interface: {bootstrap-schema--pfx}#api-admin_interface[admin_interface]
+:bootstrap-schema--xref--api-compress-responses: {bootstrap-schema--pfx}#api-compress_responses[compress_responses]
+:bootstrap-schema--xref--api-cors-headers: {bootstrap-schema--pfx}#api-cors-headers[headers]
+:bootstrap-schema--xref--api-cors-login-origin: {bootstrap-schema--pfx}#api-cors-login_origin[login_origin]
+:bootstrap-schema--xref--api-cors-max-age: {bootstrap-schema--pfx}#api-cors-max_age[max_age]
+:bootstrap-schema--xref--api-cors-origin: {bootstrap-schema--pfx}#api-cors-origin[origin]
+:bootstrap-schema--xref--api-cors: {bootstrap-schema--pfx}#api-cors[cors]
+:bootstrap-schema--xref--api-enable-advanced-auth-dp: {bootstrap-schema--pfx}#api-enable_advanced_auth_dp[enable_advanced_auth_dp]
+:bootstrap-schema--xref--api-hide-product-version: {bootstrap-schema--pfx}#api-hide_product_version[hide_product_version]
+:bootstrap-schema--xref--api-https-tls-cert-path: {bootstrap-schema--pfx}#api-https-tls_cert_path[tls_cert_path]
+:bootstrap-schema--xref--api-https-tls-key-path: {bootstrap-schema--pfx}#api-https-tls_key_path[tls_key_path]
+:bootstrap-schema--xref--api-https-tls-minimum-version: {bootstrap-schema--pfx}#api-https-tls_minimum_version[tls_minimum_version]
+:bootstrap-schema--xref--api-https: {bootstrap-schema--pfx}#api-https[https]
+:bootstrap-schema--xref--api-idle-timeout: {bootstrap-schema--pfx}#api-idle_timeout[idle_timeout]
+:bootstrap-schema--xref--api-max-connections: {bootstrap-schema--pfx}#api-max_connections[max_connections]
+:bootstrap-schema--xref--api-metrics-interface-authentication: {bootstrap-schema--pfx}#api-metrics_interface_authentication[metrics_interface_authentication]
+:bootstrap-schema--xref--api-metrics-interface: {bootstrap-schema--pfx}#api-metrics_interface[metrics_interface]
+:bootstrap-schema--xref--api-pretty: {bootstrap-schema--pfx}#api-pretty[pretty]
+:bootstrap-schema--xref--api-profile-interface: {bootstrap-schema--pfx}#api-profile_interface[profile_interface]
+:bootstrap-schema--xref--api-public-interface: {bootstrap-schema--pfx}#api-public_interface[public_interface]
+:bootstrap-schema--xref--api-read-header-timeout: {bootstrap-schema--pfx}#api-read_header_timeout[read_header_timeout]
+:bootstrap-schema--xref--api-server-read-timeout: {bootstrap-schema--pfx}#api-server_read_timeout[server_read_timeout]
+:bootstrap-schema--xref--api-server-write-timeout: {bootstrap-schema--pfx}#api-server_write_timeout[server_write_timeout]
+:bootstrap-schema--xref--api: {bootstrap-schema--pfx}#api[api]
+:bootstrap-schema--xref--bcrypt-cost: {bootstrap-schema--pfx}#bcrypt_cost[bcrypt_cost]
+:bootstrap-schema--xref--bootstrap-buckets: {bootstrap-schema--pfx}#bootstrap-buckets[buckets]
+:bootstrap-schema--xref--bootstrap-ca-cert-path: {bootstrap-schema--pfx}#bootstrap-ca_cert_path[ca_cert_path]
+:bootstrap-schema--xref--bootstrap-certpath: {bootstrap-schema--pfx}#bootstrap-certpath[certpath]
```

Comment:
Needs removing if not already in latest version.

Reply:
Done -- https://github.com/couchbase/docs-sync-gateway/pull/541